tools/linter_lib/custom_check.py

```diff
@@ -511,10 +535,12 @@ def build_custom_checkers(by_lang):
          'exclude': set(['static/templates/settings/display-settings.handlebars',
                          'templates/zerver/keyboard_shortcuts.html',
                          'templates/zerver/markdown_help.html']),
-         },
+         'good_lines': ['<img alt=("Foo Image")></img>', '<img alg=""></img>'],
```

Comment:
Same comment as above; read (or grep) to see how to do translation tags correctly in Zulip.